internal/logrus.go

```diff
@@ -22,9 +23,14 @@ type logrusLogger struct {
 	*logrus.Logger
 }
 
+type stackTracer interface {
+	StackTrace() errors.StackTrace
+}
+
 const (
 	stackTraceKey    = "stack_trace"
 	stackTraceFormat = "%s on line %d"
```

Comment:
we should probably change this to be coherent (I believe it's kind of a standard, as well) with pkg/errors' format